src/projectscene/internal/projectsceneactionscontroller.cpp

```diff
@@ -12,6 +12,7 @@ using namespace muse::actions;
 static const ActionCode VERTICAL_RULERS_CODE("toggle-vertical-rulers");
 static const ActionCode MINUTES_SECONDS_RULER("minutes-seconds-ruler");
 static const ActionCode BEATS_MEASURES_RULER("beats-measures-ruler");
+static const ActionCode PITCH_AND_SPEED_CODE("clip-pitch-speed");
```

Comment:
adding these constants arose for historical reasons, there is usually no point in them, it is even a little inconvenient to compare the action names with the handler

Reply:
renamed